src/mesh/mesh.cpp

```diff
@@ -90,18 +90,21 @@ Mesh::Mesh(ParameterInput *pin, ApplicationInput *app_in, Packages_t &packages,
       adaptive(pin->GetOrAddString("parthenon/mesh", "refinement", "none") == "adaptive"
                    ? true
                    : false),
-      multilevel((adaptive ||
-                  pin->GetOrAddString("parthenon/mesh", "refinement", "none") == "static")
-                     ? true
-                     : false),
+      multilevel(
```

Comment:
This has potential downstream implications if, for example, prolongation/restriction/check for refinement are called with a `if (multilevel)` check in the driver for a simulation without mesh refinement but with MG, doesn't it?